SimG4CMS/Calo/src/HFCherenkov.cc

```diff
@@ -17,63 +17,28 @@
 
 HFCherenkov::HFCherenkov(edm::ParameterSet const& m_HF) {
   ref_index = m_HF.getParameter<double>("RefIndex");
-  lambda1 = ((m_HF.getParameter<double>("Lambda1")) / pow(double(10), 7)) * cm;
-  lambda2 = ((m_HF.getParameter<double>("Lambda2")) / pow(double(10), 7)) * cm;
-  aperture = cos(asin(m_HF.getParameter<double>("Aperture")));
-  apertureTrap = cos(asin(m_HF.getParameter<double>("ApertureTrapped")));
-  aperturetrapped = m_HF.getParameter<double>("CosApertureTrapped");
+  lambda1 = ((m_HF.getParameter<double>("Lambda1")) / pow(double(10), 7)) * CLHEP::cm;
```

Comment:
@bsunanda , for me it is strange to use pow(double(10), 7)), instead I would put 1.e+7

Reply:
Taken care of